VideoLocker/src/main/java/org/edx/mobile/view/MyCoursesListActivity.java

```diff
@@ -37,7 +37,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public Fragment getFirstFragment() {
-        return new MyCourseListTabFragment();
+        return new MyCourseListFragment();
```

Comment:
minor nit (ignore if it isn't worth it 😉 ):
The name of the activity is My**Courses**ListActivity and the name of the fragment is My**Course**ListFragment, we should either use **Course** or **Courses** in both names.